python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -1399,28 +1702,66 @@ def is_multigraph(cls, df):
         """
         Parameters
         ----------
-        df : dataframe containing edge data
+        df : dataframe
+            Containing edge data
 
         Returns
         -------
-        True if df has >= 1 of the same src, dst pair
+        bool
+            True if df has one or more edges with the same source, destination pair
+
+        Examples
+        --------
+        >>> import cugraph
+        >>> import cudf
+        >>> from cugraph.experimental import PropertyGraph
+        >>> pG = PropertyGraph()
+        >>> df = cudf.DataFrame(columns=["src", "dst", "edge_ids" ,"some_property"],
+        ...                              data=[(99, 22, 3 , "a"),
+        ...                                    (98, 34, 5, "b"),
+        ...                                    (98, 34, 7, "c"),
+        ...                                    (96, 88, 11, "d"),
+        ...                                  ])
+        >>> pG.add_edge_data(df, type_name="etype",
+        ...                  vertex_col_names=("src", "dst"),
+        ...                  edge_id_col_name="edge_ids")
+        >>> pG.is_multigraph(pG.get_edge_data()
```

Comment:
curious if this passed doctest since it looks like this is missing a close paren.